projects/storefrontapp-e2e-cypress/cypress/helpers/product-search-pricing-flow.ts

```diff
@@ -30,30 +26,36 @@ export function productPricingFlow() {
     PRODUCT_LISTING.PRODUCTS_PER_PAGE
   );
 
-  checkFirstItem('DSC-S930');
-
-  // Navigate to next page
-  cy.get('.page-item:last-of-type .page-link:first').click();
-  cy.get('.page-item.active > .page-link').should('contain', '2');
-
-  cy.wait('@query');
-
-  cy.get('cx-product-list-item:nth-child(1)').should(
-    'contain',
-    'EASYSHARE Z730 Zoom Digital Camera'
-  );
-
-  // Sort by price low to high
-  cy.get('cx-sorting .ng-select:first').ngSelect(
-    PRODUCT_LISTING.SORTING_TYPES.BY_PRICE_ASC
-  );
-
-  cy.wait('@query_price_asc');
-
-  cy.get('.page-item.active > .page-link').should('contain', '2');
-
-  cy.get('cx-product-list-item:first .cx-product-name').should(
-    'contain',
-    'DSC-W180'
-  );
+  cy.get('cx-product-list-item .cx-product-name')
+    .first()
+    .invoke('text')
+    .should('match', /\w+/)
```

Comment:
should put a comment for the regex saying that it is taking the first word in the string

Reply:
I think that comment would be noise since this regex is very general.